src/platforms/javascript/common/install/lazy-load-sentry.mdx

```diff
@@ -26,12 +20,17 @@ To use the loader, go in the Sentry UI to _Settings -> Projects -> Client Keys (
 ></script>
 ```
 
+By default, Performance Monitoring and Session Replay is enabled. 
+
 ## Loader Configuration
 
-The loader has two configuration options:
+The loader has a few configuration options:
 
 - What version of the SDK to load
 - When to load the SDK
```

Comment:
```suggestion
```

I would take this out I think, as it's not really clear/documented how to change this...? IMHO we want to just do this however we decide makes sense (=lazy for errors only, otherwise eager).